interview-questions.md

```diff
@@ -8,58 +8,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Why would you use the method super()?
 
-  Your answer:
-
-  Researched answer:
-
+Your answer:
```

Comment:
Part of the purpose of these interview questions is to not get them right or wrong, rather it's more about developing the language and vocabulary need to talk about technical concepts. I really encourage attempting to answer the questions first --even if it's strange word vomit so that you're exercising the muscle of finding relevant words, *then* verify with researching the answer